.github/workflows/ca-renewal-system-certs-test.yml

```diff
@@ -234,6 +234,7 @@ jobs:
           docker exec pki pki \
               -u caadmin \
               -w Secret.123 \
+              --ignore-cert-status REVOKED_CERTIFICATE \
```

Comment:
Same thing, should this be `EXPIRED_CERTIFICATE`?

Reply:
As above.